Framework/test/testCustomParameters.cxx

```diff
@@ -15,73 +15,69 @@
 ///
 
 #include "QualityControl/CustomParameters.h"
-
-#define BOOST_TEST_MODULE Check test
-#define BOOST_TEST_MAIN
-#define BOOST_TEST_DYN_LINK
-
-#include <boost/test/unit_test.hpp>
 #include <boost/property_tree/ptree.hpp>
 #include <boost/property_tree/json_parser.hpp>
 #include "getTestDataDirectory.h"
+#include <iostream>
+#include <catch_amalgamated.hpp>
 
 using namespace o2::quality_control::core;
 using namespace std;
 
-BOOST_AUTO_TEST_CASE(test_cp_basic)
+TEST_CASE("test_cp_basic")
 {
   CustomParameters cp;
 
   cp.set("key", "value");
-  BOOST_CHECK_EQUAL(cp.at("key"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "default"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "default", "default"), "value");
+  CHECK(cp.at("key") == "value");
+  CHECK(cp.at("key", "default") == "value");
+  CHECK(cp.at("key", "default", "default") == "value");
 
   cp.set("key", "value_run1", "run1");
-  BOOST_CHECK_EQUAL(cp.at("key"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "run1"), "value_run1");
-  BOOST_CHECK_EQUAL(cp.at("key", "run1", "default"), "value_run1");
+  CHECK(cp.at("key") == "value");
+  CHECK(cp.at("key", "run1") == "value_run1");
+  CHECK(cp.at("key", "run1", "default") == "value_run1");
 
   cp.set("key", "value_beam1", "default", "beam1");
-  BOOST_CHECK_EQUAL(cp.at("key"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "default"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "default", "beam1"), "value_beam1");
+  CHECK(cp.at("key") == "value");
+  CHECK(cp.at("key", "default") == "value");
+  CHECK(cp.at("key", "default", "beam1") == "value_beam1");
 
   cp.set("key", "value_run1_beam1", "run1", "beam1");
-  BOOST_CHECK_EQUAL(cp.at("key"), "value");
-  BOOST_CHECK_EQUAL(cp.at("key", "run1", "beam1"), "value_run1_beam1");
+  CHECK(cp.at("key") == "value");
+  CHECK(cp.at("key", "run1", "beam1") == "value_run1_beam1");
 
   cout << cp << endl;
```

Comment:
I know that this is not part of this PR, but is the `cout` necessary in tests?

Reply:
I doubt, probably some leftover of Barth trying to see what's going on. I agree it could be removed.

Reply:
Well, perhaps one use is that we can check that operator<< does not throw, I will put it around NO_THROW statement.